src/components/Divider.tsx

```diff
@@ -54,7 +54,9 @@ const Divider = ({ path }: { path: Path }) => {
 
       /** Find and set the max width of our selected elements. */
       const maxWidth = Math.max(
-        ...Array.from(elements).map(element => (element.querySelector('.thought') as HTMLElement).offsetWidth),
+        ...Array.from(elements)
+          .filter(element => treeNode.classList.contains('table-col1') === element.classList.contains('table-col1'))
```

Comment:
`treeNode.classList.contains('table-col1')` can be precalculated outside the `filter`, or even better calculated in React:

```ts
const isTableCol1 = attributeEquals(state, head(path), '=view', 'Table')
```

Reply:
Don't worry, you previously made me aware, I didn't forget. I used DOM access since it fit with the existing approach as a quick patch to resolve #2222, which has yet to be refactored. This line would be altered along with the rest of the approach with the React-centric refactor that you had requested.

I'll get these issues resolved according to your feedback and let you know when there's an update.

Reply:
Okay, sorry for jumping the gun! Thanks.